path_config.go

```diff
@@ -154,6 +155,27 @@ func (b *kubeAuthBackend) pathConfigWrite(ctx context.Context, req *logical.Requ
 		DisableLocalCAJwt:    disableLocalJWT,
 	}
 
+	b.l.Lock()
+	defer b.l.Unlock()
+
+	// Determine if we load the local CA cert or the CA cert provided
+	// by the kubernetes_ca_cert path into the backend's HTTP client
+	certPool := x509.NewCertPool()
+	tlsConfig := &tls.Config{
+		MinVersion: tls.VersionTLS12,
+	}
+	if disableLocalJWT {
+		certPool.AppendCertsFromPEM([]byte(config.CACert))
+		tlsConfig.RootCAs = certPool
+
+		b.httpClient.Transport.(*http.Transport).TLSClientConfig = tlsConfig
+	} else {
+		certPool.AppendCertsFromPEM([]byte(localCACertPath))
```

Comment:
Have you tested this code path yet?

Reply:
Yup tested by forcing `disable_local_ca_jwt == false`  and logging what exists inside the TLS config when the auth backend is mounted. I could be mistaken but it seems like I also don't need to specify `localCACertPath` because the local CA cert is loaded into the config unless we specify we want to overwrite the CA Cert with the `kubernetes_ca_cert` path.

Reply:
What did you observe in your log output? Was the auth method able to establish a connection to k8s with the flag set to `false`?

> local CA cert is loaded into the config

Are we sure that the  (local k8s) CA cert is loaded into the config at this point in time, i.e. a cert that was not provided through `caCert`?

Reply:
> What did you observe in your log output? Was the auth method able to establish a connection to k8s with the flag set to `false`?
> 
> > local CA cert is loaded into the config
> 
> Are we sure that the (local k8s) CA cert is loaded into the config at this point in time, i.e. a cert that was not provided through `caCert`?

The CA Cert wasn't being loaded in at the time. This is why in https://github.com/hashicorp/vault-plugin-auth-kubernetes/pull/142/commits/f5470c1fbab63e24c02ab0119c959b5090c5ca49 I changed the statement to read the local CA cert and passes it into the function `AppendCertsFromPEM`. In this change, we also handle the case when the config CA cert exists so we can load the cert directly from the config.